AmplifyPlugins/Auth/Sources/AWSCognitoAuthPlugin/Models/AuthProvider+ProviderName.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import Amplify
+
+extension AuthProvider {
```

Comment:
Can we move this to Support -> Helper -> AuthProvider+Cognito? And name this identityPoolProviderName and name the one already present in AuthProvider+Cognito userPoolProviderName?